05_silly_blocks/silly_blocks.rb

```diff
@@ -0,0 +1,18 @@
+def reverser
+  yield.split(' ').map(&:reverse).join(' ')
+end
+
+def adder(num = 1)
+  yield + num
+end
+
+def repeater(num = 3)
+  yield true
+
+  n = 0
+  loop do
```

Comment:
Avoid using `loop/while` if possible. Can you find an alternate solution using `each` or `map`?